monero/utils.go

```diff
@@ -53,32 +52,3 @@ func WaitForBlocks(ctx context.Context, client WalletClient, count int) (uint64,
 		}
 	}
 }
-
-// CreateWallet creates a monero wallet from a private keypair.
-func CreateWallet(
```

Comment:
The places that were calling this now just call `CreateSpendWalletFromKeys` directly.